network/network.go

```diff
@@ -608,22 +608,28 @@ func SubnetInAnyRange(cidrs []*net.IPNet, subnet *net.IPNet) bool {
 	return false
 }
 
+// Export for testing
+var ResolverFunc = net.ResolveIPAddr
+
 // FormatAsCIDR converts the specified IP addresses to
 // a slice of CIDRs.
-func FormatAsCIDR(addresses []string) []string {
+func FormatAsCIDR(addresses []string) ([]string, error) {
 	result := make([]string, len(addresses))
 	for i, a := range addresses {
 		cidr := a
 		// If address is not already a cidr, add a /32 (ipv4) or /128 (ipv6).
 		if _, _, err := net.ParseCIDR(a); err != nil {
-			ip := net.ParseIP(a)
-			if ip.To4() != nil {
-				cidr = a + "/32"
+			address, err := ResolverFunc("ip", a)
```

Comment:
I prefer this, it parses the IP along with an attempt to resolve the address if it's a host name (although not recommended)

Reply:
gp!